templates/core/about/builds.html

```diff
@@ -3,80 +3,92 @@
+        {%- if rustc_version %}
+        The current version in use is <code>{{ rustc_version }}</code>.
+        {%- endif -%}
+    </p>
+
+    <h3>Notes on using Docs.rs</h3>
+
+    <h4>Setting a README</h4>
+    <p>
+        The README of a crate is taken from the <code>readme</code> field defined in
+        <code>Cargo.toml</code>. Omitting this field means that no README will be displayed.
+    </p>
+
+    <h4>Detecting Docs.rs from <code>build.rs</code></h4>
+    <p>
+        Docs.rs sets the environment variable <code>DOCS_RS</code> to <code>1</code>.
+        Detecting Doc.rs and building your crate differently can be helpful if you need
+        dependencies for building the library, but not for building the documentation.
+    </p>
+
+    <h4>Detecting Docs.rs from <code>#[cfg]</code> attributes</h4>
+    <p>
+        Docs.rs sets a <a href="https://doc.rust-lang.org/cargo/reference/features.html">feature</a>. See <a href="metadata">Metadata</a> for details.
+    </p>
+
+    <h4>Testing documentation builds locally</h4>
+    {%- set build_subcommand = docsrs_repo ~ "/blob/master/README.md#build-subcommand" -%}
+    <p>
+        The Docs.rs <a href="{{ build_subcommand | safe }}">README</a> describes how to build
+        unpublished crate documentation locally using the same build environment as the Docs.rs build agent.
+    </p>
+
+    <h3>Diagnosing failed builds</h3>
+
+    <h4>Missing dependencies</h4>
+    <p>
+        Missing dependencies are a common reason for failed builds.
+        Docs.rs dependencies are managed through
+        <a href="https://github.com/rust-lang/crates-build-env">crates-build-env</a>.
+        See <a href="https://forge.rust-lang.org/docs-rs/add-dependencies.html">Forge</a> for further instructions.
+    </p>
+
+    <h4>Writing files to read-only directories</h4>
+    <p>
+        Most of the sandbox is a read-only file system, including the source directory of your crate and its dependencies.
+        If you <code>build.rs</code> generates files that are relevant for documentation, consider writing to the output directory which is passed as the environment variable <code>OUT_DIR</code>.
+    </p>
+
+    <h4>Hitting sandbox limits</h4>
+    <p>
+        All the builds on Docs.rs are executed inside a sandbox with limited
+        resources. If a build fails because it hit limits please
+        <a href="{{ docsrs_repo | safe }}/issues/new/choose">open an issue</a>
+        to get them increased for your crate.
+        Since our build agent has finite resources, we have to consider each case indidually.
+        Network access will not be enabled for any crate.
+        The current limits are:
+    </p>
+
+    {{ macros::crate_limits(limits=limits) }}
+
+    <h4>Something else went wrong</h4>
+    <p>
+        Happens. Feel free to <a href="{{ docsrs_repo | safe }}/issues">file an issue</a> when having trouble.
+    </p>
```

Comment:
This section seems kind of odd. Don't we already say to open an issue above?

If we do keep it the tone should be more polite:
```suggestion
    <h4>Something else went wrong</h4>
    <p>
        Please <a href="{{ docsrs_repo | safe }}/issues">file an issue</a> if you have trouble building a crate on docs.rs.
    </p>
```